extensions/blocks/contact-form/components/jetpack-contact-form.js

```diff
@@ -190,6 +199,52 @@ class JetpackContactForm extends Component {
 		);
 	}
 
+	renderOnSubmissionFields() {
+		const {
+			customThankyou,
+			customThankyouType,
+			customThankyouMessage,
+			customThankyouRedirect,
+		} = this.props.attributes;
+		return (
+			<Fragment>
+				<ToggleControl
+					label={ __( 'Thank you message', 'jetpack' ) }
+					help={ __( 'Toggle to show a thank you message', 'jetpack' ) }
+					checked={ customThankyou }
+					onChange={ value => this.props.setAttributes( { customThankyou: value } ) }
+				/>
+				{ customThankyou && (
+					<SelectControl
+						label={ __( 'Message type', 'jetpack' ) }
+						value={ customThankyouType }
+						options={ [
+							{ label: __( 'Text message', 'jetpack' ), value: 'message' },
+							{ label: __( 'Redirect to another webpage', 'jetpack' ), value: 'redirect' },
+						] }
+						onChange={ value => this.props.setAttributes( { customThankyouType: value } ) }
+					/>
+				) }
+				{ customThankyou && ( ! customThankyouType || 'message' === customThankyouType ) && (
```

Comment:
I'm not sure why, but the `customThankyouType` attribute isn't saving in the block. Need to investigate (suggestions welcome 🙂).

Reply:
Fixed in 853e07a.